Dockerfile.local_cdk

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-slim
+
+COPY airbyte-cdk/python ./
+
+ARG CONNECTOR_NAME
+COPY airbyte-integrations/connectors/$CONNECTOR_NAME ./$CONNECTOR_NAME
```

Comment:
Just so I can understand how we can extend the usage of this: if we want to use this for Connector Builder Server for example, we would need to define another Dockerfile that copies the airbyte-cdk/python and the airbyte-connector-builder-server, right?

Reply:
That's one option, another would be to modify this file and some of the upstream code so that it works with both "normal" connectors and the connector builder server.